build.zig

```diff
@@ -468,17 +490,20 @@ fn getPaths(sdk: *Sdk, target_local: std.Build.ResolvedTarget) error{ MissingTar
             continue;
         if (target_local.result.abi != config_target.result.abi)
             continue;
-        // load paths
 
         const node = entry.value_ptr.*.object;
 
         return Paths{
-            .include = node.get("include").?.string,
-            .libs = node.get("libs").?.string,
-            .bin = node.get("bin").?.string,
+            .include = sdk.build.allocator.dupe(u8, node.get("include").?.string) catch @panic("out of memory"),
```

Comment:
Without creating dupes of these, I received the following error:

```
thread 26852 panic: @memcpy arguments alias
C:\Users\user\scoop\apps\zig\0.13.0\lib\std\fs\path.zig:111:55: 0xe7426c in joinSepMaybeZ__anon_10424 (build.exe.obj)
    @memcpy(buf[0..paths[first_path_index].len], paths[first_path_index]);
                                                      ^
C:\Users\user\scoop\apps\zig\0.13.0\lib\std\fs\path.zig:140:25: 0xe74981 in join (build.exe.obj)
    return joinSepMaybeZ(allocator, sep, isSep, paths, false);
                        ^
C:\Users\user\CLionProjects\tetris\deps\SDL.zig\build.zig:290:46: 0xec7828 in linkWindows__anon_13840 (build.exe.obj)
        const dll_path = try std.fs.path.join(sdk.build.allocator, &[_][]const u8{ paths.bin, dll_name });
                                             ^
C:\Users\user\CLionProjects\tetris\deps\SDL.zig\build.zig:361:20: 0xe78696 in link__anon_10438 (build.exe.obj)
        linkWindows(sdk, exe, linkage, library, paths) catch |err| {
                   ^
C:\Users\user\CLionProjects\tetris\deps\SDL.zig\build.zig:60:13: 0xe75223 in build (build.exe.obj)
    sdk.link(demo_wrapper, sdl_linkage, .SDL2);
            ^
C:\Users\user\scoop\apps\zig\0.13.0\lib\std\Build.zig:2117:43: 0xe5d9e2 in runBuild__anon_8979 (build.exe.obj)
        .ErrorUnion => try build_zig.build(b),
                                          ^
C:\Users\user\scoop\apps\zig\0.13.0\lib\compiler\build_runner.zig:301:29: 0xe58d04 in main (build.exe.obj)
        try builder.runBuild(root);
                            ^
C:\Users\user\scoop\apps\zig\0.13.0\lib\std\start.zig:363:53: 0xe5fc1c in WinStartup (build.exe.obj)
    std.os.windows.ntdll.RtlExitUserProcess(callMain());
                                                    ^
???:?:?: 0x7ff86786257c in ??? (KERNEL32.DLL)
???:?:?: 0x7ff8692eaf27 in ??? (ntdll.dll)
error: unable to read results of configure phase from 'C:\Users\user\CLionProjects\tetris\deps\SDL.zig\.zig-cache\tmp\1df158d523678d51': FileNotFound
```

Maybe the variables passed to the `sdk.build.installBinFile` are being deinitialized too early.